scripts/globals/conquest.lua

```diff
@@ -58,7 +58,8 @@ end
 -- (LOCAL) outposts
 -----------------------------------
 
-local outposts = {
+local outposts =
```

Comment:
> they can't be teleported to, but are considered for conquest tally

actually tulia has a legit teleport it just goes unused on retail. but the NPC event for outpost tele's will display and use it (and we gave it a pos for warpage on DSP enabled by a setting I think).